benchmarks/dynamo/common.py

```diff
@@ -130,6 +130,109 @@ class CI(NamedTuple):
     "dlrm",
 }
 
+# These models currently fail accuracy with eager Adam optimizer
```

Comment:
Could you open an issue tracker and link here in the comment?

Reply:
It is really hard to maintain this kind of list unless you are actively working on shrinking it. I finally got rid of the CI_SKIP list and moved to comparing against the expected result files, so it's sad to see another list like this. If we add another set of expected result files for `Adam`, I wonder how much test time that would add?

Reply:
So are you suggesting doing a whole other benchmark run with Adam with a separate set of result files? I think that would add a lot of time right?

FWIW, I am planning on shrinking this list next half, because we should be running Adam on all models IMO.